src/kirby/components/list/list.component.tns.scss

```
@@ -1,11 +1,8 @@
 @import "../../scss/native-imports";
-$list-item-height: 44;
-$list-item-padding: 16;
-$list-header-height: 38;
 .row {
   padding: 16 8 16 8;
```

Should this be a theming size? 😄 I am not sure if you need to specify all padding or only need left/right and top..
```suggestion
  padding: size('gutter') size('gutter-small') size('gutter') size('gutter-small');
```

---

Thanks didn't know we had cutter sizes.